tests/integration/AspNet.Core.Tests/TestBot.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.Tests
+{
+    internal sealed class TestBot : IBot
```

Comment:
So the point of having it throw the exception is so that nobody else writes a test that tries to use it for something it's not meant for. I'll put a better message in the exception to clarify this.